docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
             context: frontend/
             dockerfile: Dockerfile
         restart: always
-        command: ['sh', '-c', 'if [ ! -d "node_modules" ]; then yarn install; fi; yarn start > /var/log/app.log 2> /var/log/error.log']
+        command: ['sh', '-c', 'if [ ! -d "node_modules" ]; then yarn install; fi; yarn start > /dev/stdout 2>&1']
```

Comment:
don't think you need the 2>&1, as it worked fine for me without.

Reply:
just making sure to log stderr out to docker too.